### PR TITLE
Add Conrad

### PR DESCRIPTION
**Pull request by jokay**
Adds [Conrad](https://www.conrad.ch/).

rank-conrad

### CODE REVIEW

**Comment by phallobst** [comment]
Since Conrad is predominantly a German company, I guess we should use https://conrad.de instead, maybe with the `regions` as well - although there are quite a number of them according to https://www.conrad.de/de/ueber-conrad/unternehmen.html.

**Comment by phallobst** [comment]
@Carlgo11 The Facebook check is currently broken, since the header does not include `location` anymore. :frowning_face:

**Comment by Carlgo11** [comment]
@phallobst Nooooo! 😭
I'll try and fix it tonight

File: _data/retail.yml [hunk 1]
```diff
@@ -110,6 +110,31 @@ websites:
...
+      - at
+      - pl
+      - ro
+      - se
+      - ch
+      - sk
+      - si
+      - cz
+      - hu
+      - uk
```
Comment (phallobst) [inline _data/retail.yml]:
```suggestion
      - gb
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Comment by phallobst** [comment]
Facebook is messing around currently, I guess. Right now the headers include `Location` (capital L), but a misspelled handle is not corrected.
_Edit_: And now the check is working again?! I have no idea what is happening...

**Comment by jokay** [comment]
> Facebook is messing around currently, I guess. Right now the headers include `Location` (capital L), but a misspelled handle is not corrected.
> _Edit_: And now the check is working again?! I have no idea what is happening...

Conrad Switzerland has `conrad.schweiz` (with `.`) and Conrad Germany `conradelectronic` as Facebook account, this is why it may work again :wink:

**Review by Carlgo11 — APPROVED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**